src/java.base/share/classes/jdk/crac/CheckpointException.java

```diff
@@ -38,4 +38,14 @@
     public CheckpointException() {
         super();
     }
+
+    /**
+     * Constructs a {@code CheckpointException} with the specified
+     * detail message.
+     *
+     * @param   message   the detail message.
```

Comment:
I see these extra spaces in other javadocs https://github.com/openjdk/jdk/blob/716c2e1e85c244b31cf73e5ac1a2d765e3514c4d/src/java.base/share/classes/java/lang/LinkageError.java#L52

Are there reasons for this? (The same on line 62 in the linked file makes sense -- alignment of parameter descriptions)

Reply:
It was just a copy past from superclass. Extra spaces are removed